src/applications/benefit-eligibility-questionnaire/components/BenefitCard.jsx

```diff
@@ -2,10 +2,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const BenefitCard = ({ benefit }) => {
-  const { name, category, description, learnMoreURL, applyNowURL } = benefit;
+  const {
+    name,
+    category,
+    description,
+    isTimeSensitive,
```

Comment:
It looks like the new code at line 9 introduces a new linting warning. Can we please clean this up before we merge this?

Reply:
Ok, this is fixed.